src/app/shared/user.service.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { TestBed } from '@angular/core/testing';
+import { UserService } from './user.service';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { Observable } from 'rxjs';
+
+fdescribe('UserService', () => {
```

Comment:
use describe instead of fdescribe. fdescribe will run only that test case nothing else.

Reply:
done